### PR TITLE
[Segment] Capture SSH_TTY env

### LINKED ISSUE #2204: Detect if CRC is running in a remote server via an ssh session
using $SSH_TTY we can detect people who ssh to linux server and start crc there

/cc @veillard

### PR DESCRIPTION
**Pull request by praveenkumar**
This patch captures following scenario and set true for `SSH_TTY` which
able to provide us bit detail around how many user are using crc on
remote host.
- User ssh to a remote box
- User run `crc` commands from that box

fixes #2204

### CODE REVIEW

**Comment by jsliacan** [comment]
@praveenkumar , maybe we should also add it to the [doc](https://github.com/code-ready/crc/blob/master/usage-data.adoc)? Thanks!

File: pkg/os/util.go [hunk 1]
```diff
@@ -115,3 +115,7 @@ func RemoveFileIfExists(path string) error {
 func RunningInTerminal() bool {
 	return terminal.IsTerminal(int(os.Stdin.Fd()))
 }
+
+func RunningUsingSSH() bool {
+	return os.Getenv("SSH_TTY") != ""
```
Comment (cfergeau) [inline pkg/os/util.go:120]: Or `LookupEnv`? I guess `SSH_TTY` is not going to be set but empty, but `LookupEnv` seems more correct

File: usage-data.adoc [hunk 2]
```diff
@@ -20,6 +20,7 @@ The following table describes the opt-in usage data collected by CodeReady Conta
 |                  | Experimental features       | true/false
 |                  | tty                         | true/false
 |                  | duration                    |
+|                  | remote (Capture $SSH_TTY)   | true/false
```
Comment (cfergeau) [inline usage-data.adoc:23]: It captures if it's set or not, not the value of `$SSH_TTY`. Maybe just `remotely connected with SSH`, how it's implemented does not necessarily have to be mentioned.

File: pkg/os/util.go (hunk 1 above)
Comment (praveenkumar) [reply pkg/os/util.go:120]: @cfergeau It is going to be set
```
// On my current workstation
$ env | grep -i ssh
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
SSH_AGENT_PID=5205


// On the remote machine
$ env | grep -i ssh
SSH_CONNECTION=10.10.113.37 45514 10.0.78.252 22
SSH_CLIENT=10.10.113.37 45514 22
SSH_TTY=/dev/pts/0
```

File: usage-data.adoc (hunk 2 above)
Comment (praveenkumar) [reply usage-data.adoc:23]: Sure, I can change the wording.

Comment (jsliacan) [reply usage-data.adoc]: Then maybe also de-mystify the `tty` above? Like @cfergeau says, let's just say what we find out with it (in plain English) -- whether CRC is used in a terminal or not?.

Comment (praveenkumar) [reply usage-data.adoc]: @jsliacan not in this PR.

**Review by anjannath — APPROVED** [review]

**Review by cfergeau — APPROVED** [review]
